bentoml/utils/usage_stats.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2019 Atalaya Tech, Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+
+import sys
+import platform
+import json
+
+import uuid
+import requests
+
+from bentoml import config, load
+import bentoml
+
+AMPLITUDE_URL = "https://api.amplitude.com/httpapi"
+API_KEY = '7f65f2446427226eb86f6adfacbbf47a'
+
+
+def get_artifact_handler_info(bento_service):
+    artifact_types = []
+    handler_types = []
+
+    for artifact in bento_service.artifacts.items():
+        artifact_instance = artifact[1]
+        artifact_types.append(artifact_instance.__class__.__name__)
+
+    for api in bento_service.get_service_apis():
+        handler_types.append(api.handler.__class__.__name__)
+
+    return {"handler_types": handler_types, "artifact_types": artifact_types}
+
+
+def track(event_type, info):
+    info["py_version"] = "{major}.{minor}.{micro}".format(
```

Comment:
move this and `bento_version`, `platform` to top of this file as constants?

Reply:
done